src/index.js

```diff
@@ -1,12 +1,55 @@
 import './styles/style.css';
-import UI from './modules/ui.js';
-import Todo from './modules/todo.js';
 
-document.addEventListener('DOMContentLoaded', UI.displayTodo);
+import updateTaskStatus from './modules/task.js';
+import {
+  displayTask, addTask, deleteTask, lists,
+} from './modules/all.js';
 
-// eslint-disable-next-line no-undef
+const form = document.getElementById('add-to-list');
+const btnClear = document.querySelector('.btn-clear');
 
-// eslint-disable-next-line no-undef, no-unused-vars
-const book = new Todo(desc, completed, id);
+const editTask = (task) => {
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
+  const taskItem = task.children[0].children[1].name;
+  const taskIndex = tasks.findIndex((x) => x.description === taskItem);
+  const taskName = task.querySelector('#task-name').value;
+  tasks[taskIndex].description = taskName;
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
 
-UI.addBookToList(Todo);
\ No newline at end of file
+lists.addEventListener('click', (e) => {
+  const task = e.target.parentElement.parentElement;
+  if (e.target.classList.contains('del')) {
+    deleteTask(task, e.target);
+  }
+
+  if (e.target.classList.contains('edit')) {
+    editTask(task);
```

Comment:
- [OPTIONAL] All the actions are working and the edit functionality too, but it would be great if the edit functionality happens right after the user makes a change to the input like adding or removing words so he/she doesn't need to click on the button to record the changes.